File: Tests/NyxCoreTests/Fixtures/export/01-chrome-copy-as-curl.go
package main

import (
	"fmt"
	"io"
	"net/http"
	"strings"
)

func main() {
	req, err := http.NewRequest("POST", "https://api.example.com/v1/messages", strings.NewReader(`{"model":"claude-opus","stream":true,"messages":[{"role":"user","content":"hi
there"}]}`))
	if err != nil {
		panic(err)
	}
	req.Header.Set("accept", "*/*")
	req.Header.Set("accept-language", "en-US,en;q=0.9")
	req.Header.Set("cache-control", "no-cache")
	req.Header.Set("content-type", "application/json")
	req.Header.Set("origin", "https://app.example.com")
	req.Header.Set("pragma", "no-cache")
	req.Header.Set("priority", "u=1, i")
	req.Header.Set("referer", "https://app.example.com/")
	req.Header.Set("sec-ch-ua", "\"Chromium\";v=\"128\", \"Not;A=Brand\";v=\"24\"")
	req.Header.Set("sec-ch-ua-mobile", "?0")
	req.Header.Set("sec-ch-ua-platform", "\"macOS\"")
	req.Header.Set("sec-fetch-dest", "empty")
	req.Header.Set("sec-fetch-mode", "cors")
	req.Header.Set("sec-fetch-site", "same-site")

	client := &http.Client{}
	resp, err := client.Do(req)
	if err != nil {
		panic(err)
	}
	defer resp.Body.Close()

	body, _ := io.ReadAll(resp.Body)
	fmt.Println(resp.StatusCode, string(body))
}

// not translated: --compressed

File: Tests/NyxCoreTests/Fixtures/export/01-chrome-copy-as-curl.py
import requests

response = requests.request(
    "POST",
    "https://api.example.com/v1/messages",
    headers={
        "accept": "*/*",
        "accept-language": "en-US,en;q=0.9",
        "cache-control": "no-cache",
        "content-type": "application/json",
        "origin": "https://app.example.com",
        "pragma": "no-cache",
        "priority": "u=1, i",
        "referer": "https://app.example.com/",
        "sec-ch-ua": "\"Chromium\";v=\"128\", \"Not;A=Brand\";v=\"24\"",
        "sec-ch-ua-mobile": "?0",
        "sec-ch-ua-platform": "\"macOS\"",
        "sec-fetch-dest": "empty",
        "sec-fetch-mode": "cors",
        "sec-fetch-site": "same-site",
    },
    json={
        "model": "claude-opus",
        "stream": True,
        "messages": [
            {
                "role": "user",
                "content": "hi\nthere",
            },
        ],
    },
)

print(response.status_code, response.text)
# not translated: --compressed
